lms/djangoapps/shoppingcart/processors/CyberSource.py

```diff
@@ -450,3 +450,10 @@ def get_processor_exception_html(exception):
             """)),
     }
 )
+
+
+def sychronize_transactions(start_date, end_date):  # pylint: disable=unused-argument
+    """
+    Concrete implementation of the API
+    """
+    raise Exception("Not implemented")
```

Comment:
Python has a built in [`NotImplementedError`](https://docs.python.org/2/library/exceptions.html#exceptions.NotImplementedError) exception class -- you probably want to use that, instead of the generic `Exception` class.